src/SprykerEco/Client/Adyen/AdyenClient.php

```diff
@@ -66,4 +66,21 @@ public function handleCreditCard3dResponseFromAdyen(AdyenRedirectResponseTransfe
             ->createZedAdyenStub()
             ->handleCreditCard3dResponseFromAdyen($redirectResponseTransfer);
     }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\AdyenRedirectResponseTransfer $redirectResponseTransfer
+     *
+     * @return \Generated\Shared\Transfer\AdyenRedirectResponseTransfer
+     */
+    public function handleKlarnaInvoiceResponseFromAdyen(AdyenRedirectResponseTransfer $redirectResponseTransfer): AdyenRedirectResponseTransfer
+    {
+        return $this
```

Comment:
Doesn't look good as for me. It's better to use `return $this->getFactory()` and after that break the line.

Reply:
I wrote code in the same style of that class other methods. Should I change code to your variant?